pagopa/io-app
# feat: [IA-312] Add PayPal card wallet preview

## Short description
This PR adds the PayPal wallet card preview

| contained text  | overflow text |
| ------------- | ------------- |
![a](https://user-images.githubusercontent.com/822471/139069399-4c586c89-d618-4852-ae52-f6179075ca61.png) | ![b](https://user-images.githubusercontent.com/822471/139092252-c8cc8943-3bf9-45fb-a604-c3641beeee3e.png)


## How to test
- replace some of existing payment method with this new one in `ts/features/wallet/component/card/WalletV2PreviewCards.tsx`
example

```typescript
const paymentMethodPreview = (pm: PaymentMethod): React.ReactElement | null => {
  switch (pm.kind) {
    case "Satispay":
      return <SatispayWalletPreview key={pm.idWallet} satispay={pm} />;
    case "Bancomat":
      return (
        <PayPalWalletPreview
          paypal={{ email: "ma***ma***ma***ma***ma***ma***@gmail.com" }}
        />
      );
    case "CreditCard":
      // We should distinguish between a plain credit card and a cobadge credit card.
      // Unfortunately, the cobadge card doesn't have a own type but is a CreditCard that has an issuerAbiCode
      return isCobadge(pm) ? (
        <CobadgeWalletPreview key={pm.idWallet} cobadge={pm} />
      ) : (
        <CreditCardWalletPreview key={pm.idWallet} creditCard={pm} />
      );
    case "BPay":
      return <BPayWalletPreview key={pm.idWallet} bPay={pm} />;
    case "Privative":
      return <PrivativeWalletPreview key={pm.idWallet} privative={pm} />;
  }
};
```
